Assets/MRTK/SDK/Experimental/MixedRealityKeyboard/MixedRealityKeyboardBase.cs

```diff
@@ -1,38 +1,25 @@
+//#define WINDOWS_UWP
```

Comment:
Why is this line here? It is commented, so is safe, but the code should get platform defines from the project.

Reply:
Good catch!